tests/integ/test_stored_procedure_profiler.py

```diff
@@ -256,6 +258,46 @@ def test_create_temp_stage(profiler_session):
         profiler_session.sql(f"use database {current_db}").collect()
 
 
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="session.sql is not supported in localtesting",
+)
+def test_stored_proc_error(
+    is_profiler_function_exist, profiler_session, db_parameters, tmp_stage_name
+):
+    profiler_session.sql('alter session set PYTHON_UDF_CGROUP_MEMSIZE="1g"').collect()
+    profiler_session.sql("alter session set UDF_SET_CGROUP_ENABLE = true").collect()
+    profiler_session.sql("alter session set UDF_CGROUP_ENABLE = true").collect()
+    profiler_session.sql("ALTER SESSION SET ENABLE_UDF_OOM_NOTIFIER = TRUE;").collect()
+    function_name = f"oom_sp_{Utils.random_function_name()}"
+
+    @sproc(name=function_name, replace=True)
+    def oom_sp(session: snowflake.snowpark.Session) -> str:
+        gb = 1024 * 1024 * 1024
+        x = "*" * gb * 5
+        return f"Return string is of length {len(x)}"
+
+    profiler_session.stored_procedure_profiler.register_modules(["oom_sp"])
+    profiler_session.stored_procedure_profiler.set_target_stage(
+        f"{db_parameters['database']}.{db_parameters['schema']}.{tmp_stage_name}"
+    )
+
+    profiler_session.stored_procedure_profiler.set_active_profiler("LINE")
+
+    with pytest.raises(
+        SnowparkSQLException, match="Function available memory exhausted"
+    ):
+        profiler_session.call(function_name)
+    res = profiler_session.stored_procedure_profiler.get_output()
+
+    profiler_session.stored_procedure_profiler.disable()
+
+    profiler_session.stored_procedure_profiler.register_modules()
+
+    assert res is not None
+    assert "oom_sp" in res
```

Comment:
what's the type of res?
I feel you can merge two lines into one line as it's impossible that "oom_sp" in None stands
```suggestion
    assert "oom_sp" in res
```

Reply:
it is a string, will do